msteams-platform/toolkit/VS-TeamsFx-preview-and-customize-app-manifest.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Teams App Manifest in Teams Toolkit for Visual Studio
+author: surbhigupta
+description: In this module, learn how to edit, preview and customize Teams App Manifest in the different environment for Visual Studio.
+ms.author: nliu
+ms.localizationpriority: medium
+ms.topic: overview
+ms.date: 05/13/2022
+---
+
+# Edit Teams app manifest using Visual Studio
+
+Teams Toolkit in Visual Studio loads manifest from `manifest.template.json` with configurations from `state.{env}.json` and `config.{env}.json` while provisioning and preparing app dependencies. This allows you to create Microsoft Teams app in [Developer Portal](https://dev.teams.microsoft.com/apps) with the manifest.
+
+After scaffolding, in the manifest template file under `templates/appPackage` folder,
+`manifest.template.json` is shared between local and remote environment.
```

Comment:
this part sounds confusing to me :( do you actually mean the manifest file `manifest.template.json` under `templates/appPackage` folder is shared between local and remote environment?

Reply:
The information has been updated as per the draft received: https://github.com/OfficeDev/teamsfx-docs/pull/33/files
Hi @zyxiaoyuer, please help us unblock here.

Reply:
Yes, the template file is shared by all environments.